test/autoyast_converter_test.rb

```diff
@@ -93,7 +93,7 @@
     end
   end
 
-  describe ".export" do
+  xdescribe ".export" do
```

Comment:
It should be quite easy to implement something like that using doubles. Like here, for example, where we simulate a devicegraph with two disks.
https://github.com/yast/yast-storage-ng/blob/master/test/support/boot_requirements_context.rb#L61